plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -42,7 +42,7 @@
 DBT_VERSION_TUPLE = (int(DBT_VERSION.major), int(DBT_VERSION.minor))
 
 if DBT_VERSION_TUPLE >= (1, 0):
-    from dbt.flags import PROFILES_DIR
+    from dbt.flags import PROFILES_DIR  # noqa: F811
```

Comment:
No longer need this `noqa` since removing above import.

```suggestion
    from dbt.flags import PROFILES_DIR
```